### DISCUSSION TITLE
about parameter

### DISCUSSION THREAD

**Discussion by vipera666** [esnet/iperf#1630] [General]
Friends,

i need a parameter to test an especific bandwith, is it possible?Cause i use iperf with no specification, the test freezes cause link saturation, so, i d like to test with a parameter that use not all the bandwith.For example, i have a 40 mbps link, may i test this using only 35 mbps?
thanks

**Comment by davidBar-On** [comment]
Hi, use `iperf3 -h` to see the available parameters.  Regarding bandwidth limit, use `-b`.  E.g. `-b 35M` limit the bandwidth to 35Mbps.
